src/Elasticsearch/Query.php

```diff
@@ -66,8 +66,12 @@ public function find() : array
         curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
         curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
 
+        $results = curl_exec($ch);
+        if (!$results) {
+            throw new Exception('Error connecting to ElasticSearch Node - Please your connection host');
```

Comment:
@FredPeal lets throw the exception based on the curl error 
https://stackoverflow.com/a/3987037/10697357